### DISCUSSION TITLE
Support for the others

### DISCUSSION THREAD

**Discussion by CodeLonelyHeart** [Detroit-Become-Human-Readable/Audio#1] [Ideas]
Could you support for other Quantic Dream games, like HEAVY RAIN,Beyond two souls,Fahrenheit: Indigo Prophecy
These game's music are beautiful

**Comment by root-mega** [comment]
neither of us have any of the games, but it sounds like something that could be possible in the future

**Reply by TranscientNight-exe** [reply]
If you finish this current project, will DBH sound effects become available? Such as interface/gameplay sound effects (ex click or mind palace sounds) or sfx from the game (ex gunshots, environmental  noises)
